apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/deprecated/AnyAssertionSamples.kt

```diff
@@ -50,4 +61,188 @@ class AnyAssertionSamples {
+            //                              | fails
+        }
+    }
+
+    @Test
+    fun notToBeNull() {
+        expect<Int?>(1) notToBeNull { // subject is now of type Int
+            it isGreaterThan 0
+            it isLessThan 10
+        } toBe 1 // subject here is remains to type Int
+
+        fails {
+            // because you forgot to define an assertion in the assertion group block
+            // use `notToBeNull()` if this is all you want to assert
+            expect<Int?>(1) notToBeNull { }
+        }
+
+        fails {
+            // notToBeNull already fails, reporting mentions that subject was expected `to be: 2`
+            expect<Int?>(null) notToBeNull { // subject inside this block is of type Int (actually 1)
+                it toBe 2
+            }  // subject here is remains to type Int
+        }
+
+        fails {
+            expect<Int?>(1) notToBeNull { // subject inside this block is of type Int (actually 1)
+                it isLessThan 0
+            } // subject here is remains to type Int
+        }
+    }
+
+    @Test
+    fun isAFeature() {
+        val n: Number = 1
+        expect(n).isA<Int>() isGreaterThan  0
+        //          | subject is now of type Int
+
+        fails {
+            expect("A").isA<Long>() isLessThan 2L // not shown in reporting as `isA<Long>()` already fails
+        }
+    }
+
+    @Test
+    fun isA() {
+        val n: Number = 16
+        expect(n).isA<Int> { // subject is now of type Int, within this block but also afterwards
+            it isGreaterThanOrEqual 15
+        } isLessThan 20
+
+        fails {
+            // because wrong type expected (Long instead of String)
+            expect("A").isA<Long> {
+                it toBe 43
+            }
+        }
+
+        fails {
+            // type fits, but sub-assertion fails
+            expect(54L).isA<Long> {
+                it toBe -1L
+            }
+        }
+    }
+
+    @Test
+    fun andFeature() {
+        // and is just a filler word does not have any behaviour
+        expect(13) isGreaterThan 5 and o isLessThan 20
+
+        // i.e. the above is equivalent to:
+        expect(13) isGreaterThan 5 isLessThan 20
+    }
+
+    @Test
+    fun and() {
+        expect(13).isA<Int>() and {
+            it isGreaterThan 5
+            it isLessThan 20
+        }
+
+        fails {
+            expect(13).isA<Int>() and {
+                // all assertions are evaluated inside an assertion group block; for more details:
+                // https://github.com/robstoll/atrium#define-single-assertions-or-assertion-groups
+                it isNoneOf values(1, 2, 13)  // fails
+                it isLessThan 10  //                                 still evaluated and included in the error report
+                //                                                   use `.and.` if you want fail fast behaviour
```

Comment:
```suggestion
                it isNoneOf values(1, 2, 13)  // fails
                it isLessThan 10              // still evaluated and included in the error report
                                              //  use `.and.` if you want fail fast behaviour
```

Reply:
I did this way because if you hit autofromat in IntelliJ then it idents the comments to the left, but this way it stays at the same place. Should I change it to look like your suggestion?

Reply:
I see. That's fine as well. Is it GitHub are the two text too far to right?

Reply:
It's GitHub, in the source code the 3 lines starts at the same position.

Reply:
Looks the same for me like on github. Are you using a monospaced font?

Reply:
Yes, JetBrains Mono. I'll try to checkout the repo again and see what it looks like.

Reply:
You were right. I don't know why IntelliJ shows this way, but i fixed it in VIM. See #864 .